src/main/common/momentFactory.ts

```diff
@@ -6,7 +6,7 @@ export class MomentFactory {
   }
 
   static currentDate (): moment.Moment {
-    return moment().hours(0).minutes(0).seconds(0).milliseconds(0)
+    return moment.utc().hours(0).minutes(0).seconds(0).milliseconds(0)
```

Comment:
That is not safe IMO. It means all dates will be sent to backend as UTC e.g. (2018-03-26T10:37:42.851+00:00), then if that is mapped to `LocalDateTime` on the backend side we would store `2018-03-26T10:37:42.851+00:00` instead of `2018-03-26T11:37:42.851+00:00`

UTC is safe for our custom local date class but not for everything.

Reply:
This method just handles date not with Time, are all mapped against LocalDate